Sources/Commands/PackageTools/Describe.swift

```diff
@@ -26,7 +26,7 @@ extension SwiftPackageTool {
         @OptionGroup(visibility: .hidden)
         var globalOptions: GlobalOptions
         
-        @Option(help: "json | text")
+        @Option(help: "json | text | mermaid")
```

Comment:
Could this be "output format"? Newer versions of argument parser should handle printing the valid options.

Reply:
Yes, I can do that, but we have to update to a new version of ArgumentParser first, and that's a quite complicated process that involves a coordinated set of PRs across all of toolchain repositories that depend on it. Seemed out of scope for this specific PR.

Reply:
I did not know that! We can make an issue for modernizing the argument helps whenever the arg parser is bumped next.

Reply:
https://github.com/apple/swift-package-manager/issues/7290